app/src/androidTest/java/org/simple/clinic/di/TestAppComponent.kt

```diff
@@ -26,6 +26,7 @@ import org.simple.clinic.facility.FacilityRepositoryAndroidTest
 import org.simple.clinic.login.LoginUserWithOtpServerIntegrationTest
 import org.simple.clinic.medicalhistory.MedicalHistoryRepositoryAndroidTest
 import org.simple.clinic.monthlyReports.questionnaire.QuestionnaireRepositoryAndroidTest
+import org.simple.clinic.monthlyReports.questionnaire.QuestionnaireResponseRepositoryAndroidTest
```

Comment:
I know we discussed fixing it towards the end, but this is driving me crazy a bit.
@siddh1004 let's create a separate story for fixing namespaces? I have 3 issues with current heriarchy:
1. `monthlyReports` is a parent for generic things
2. There's redundancy of `questionnaire`. For ex: `questionnaire.QuestionnaireRepositoryAndroidTest`
3. `Questionnaire` & `QuestionnaireResponses` are 2 separate things nested inside 1 same parent `questionnaire`

Reply:
@olttwa I will create user stories for this.